yb-voyager/cmd/cutover.go

```diff
@@ -45,31 +49,44 @@ func init() {
 		"wait for the cutover to complete")
 }
 
-func InitiatePrimarySwitch(action string) {
-	createTriggerIfNotExists(getTriggerName(action))
+func InitiatePrimarySwitch(action string) error {
+	triggerName, err := getTriggerName(action)
+	if err != nil {
+		return fmt.Errorf("failed to get trigger name: %w", err)
```

Comment:
Nit: while wrapping error messages using `Errorf` we should not phrase the message something as `failed to perform action: %w`. Instead, we should just say `action: %w`. That will make the error message look like a semantic stack trace e.g. `action1: action2: action3: original error message`.

In this case, one could just write `fmt.Errorf("get trigger name: %w", err)`.

Reply:
updated at all the relevant places.